### DISCUSSION TITLE
Make the copyable and viewable input actions accessible for screenreaders

### DISCUSSION THREAD

**Discussion by s-tinga** [livewire/flux#835] [Feature Requests]
Currently if you add the `copyable` or `viewable` properties to an input the button that Flux generates is not accessible to screenreaders. It's just an empty button with an icon in it which screenreaders can't read. I've worked around this by publishing the input component and manually adding some `sr-only` text within the buttons for screenreaders to reference, but ideally Flux would be able to handle this and be fully accessible without requiring me to publish components and edit them!

The `clearable` action has an `aria-label` on it, but the other two don't have any kind of aria labels or `sr-only` text. 

e.g. `flux/input/copyable.blade.php`
My screenreader sees this as just "button" which gives no clues as to what this action actually does if I choose to interact with it. 
```html
<flux:button
    :$attributes
    :size="$size === 'sm' ? 'xs' : 'sm'"
    x-data="{ copied: false }"
    x-on:click="copied = ! copied; navigator.clipboard && navigator.clipboard.writeText($el.closest('[data-flux-input]').querySelector('input').value); setTimeout(() => copied = false, 2000)"
    x-bind:data-copyable-copied="copied"
>
    <flux:icon.clipboard-document-check variant="mini" class="hidden [[data-copyable-copied]>&]:block" />
    <flux:icon.clipboard-document variant="mini" class="block [[data-copyable-copied]>&]:hidden" />
</flux:button>

```

I manually published and adjusted this to ad `sr-only` text allows this button to have a label for screenreaders so it reads something like "Clear input, button" instead which makes a lot more sense. 
```html
<flux:button
    :$attributes
    :size="$size === 'sm' ? 'xs' : 'sm'"
    x-data="{ copied: false }"
    x-on:click="copied = ! copied; navigator.clipboard && navigator.clipboard.writeText($el.closest('[data-flux-input]').querySelector('input').value); setTimeout(() => copied = false, 2000)"
    x-bind:data-copyable-copied="copied"
>
    <div class="hidden [[data-copyable-copied]>&]:block">
        <flux:icon.clipboard-document-check variant="mini" />
        <span class="sr-only">Copied to clipboard</span>
    </div>

    <div class="block [[data-copyable-copied]>&]:hidden">
        <flux:icon.clipboard-document variant="mini" />
        <span class="sr-only">Copy to clipboard</span>
    </div>
</flux:button>
```

**Comment by joshhanley** [comment]
@stuininga-pl thanks for reporting! This to me seems more like a bug/ issue than a feature request as Flux is designed to be accessible out of the box. Can I get you to please open this as a new issue and include all the details in the issue so it can be investigated? Thanks!
